src/viewport/index.tsx

```diff
@@ -547,8 +551,8 @@ class Viewport extends React.Component<
             }
             if (!this.props.lockedCamera) {
                 this.visGeometry.setFollowObject(intersectedObject);
+                this.visGeometry.addPathForAgent(intersectedObject);
```

Comment:
this is just moved within the block to prevent a path from being render if the user click on an agent while the camera is disabled